tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/data/PetData.java

```diff
@@ -13,14 +13,12 @@
 
 package org.eclipse.microprofile.openapi.apps.petstore.data;
 
+import org.apache.commons.lang3.StringUtils;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Category;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Pet;
 import org.eclipse.microprofile.openapi.apps.petstore.model.Tag;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.HashMap;
+import java.util.*;
```

Comment:
do we really need to import everything from that package?

Reply:
Definitely not. :)

Reply:
Hi, I'm so sorry. It is not needed.

Reply:
I'll fix it.

Reply:
Updated.